README.md

```diff
@@ -135,9 +135,7 @@ oTracking.page(pageConfig);
 
 Call the `oTracking.click.init` method to track click events. Pass the category you would like the click events to have as an argument:
 
-- If the element being clicked is a link which goes to a page on the same domain as the current page, o-tracking will put the tracking data into a queue to send to Spoor at a later time.
-- Add the attribute `data-o-tracking-skip-queue` to the element to send the data to Spoor immediately.
```

Comment:
Is the plan to release as a new major version due to the removal of this feature?